proc_stat_io.go

```diff
@@ -0,0 +1,58 @@
+package procfs
+
+import (
+	"fmt"
+	"io/ioutil"
+)
+
+type ProcIO struct {
+	// chars read
+	rchar int64
+	// chars written
+	wchar int64
+	// read syscalls
+	syscr int64
+	// write syscalls
+	syscw int64
+	// bytes read
+	read_bytes int64
```

Comment:
Instead of under_scores in Go, please use camelCase.

And yes, these fields need to be exported (capitalized first letter), as otherwise they won't be visible outside of the package.